repository.rst

```diff
@@ -214,21 +214,6 @@ returned by ``go env GOPATH``.
 | mode, Gazelle will run if there is no build file in the repository root                                 |
 | directory.                                                                                              |
 +--------------------------------+----------------------+-------------------------------------------------+
-| :param:`build_config`          | :type:`label`        | :value:`@//:WORKSPACE`                          |
```

Comment:
Looking for suggestions on the best way to document this functionality

Reply:
I think we should keep `build_config` and update the second paragraph to say that `go_repository` rules are only re-evaluated when parts of WORKSPACE related to Gazelle's configuration are changed, including Gazelle directives and `go_repository` `name` and `importpath` attributes.